autogen/main/variables.tf.tmpl

```diff
@@ -858,6 +858,13 @@ variable "enable_shielded_nodes" {
 }
 
 {% endif %}
+
+variable "default_compute_class_enabled" {
+  type        = bool
+  description = "Enable Spot VMs as the default compute class for Node Auto-Provisioning"
+  default     = null
+}
+
```

Comment:
description is misleading, no? I believe this has nothing to do with SPOT vms?

https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#default_compute_class_enabled-1

@rs1986x @apeabody

Reply:
that's a good point - yeah... that is the way I use it, it is not the correct description.